src/core_plugins/kibana/common/tutorials/functionbeat_instructions.js

```diff
@@ -0,0 +1,468 @@
+        values: {
+          passwordTemplate: '`<password>`',
+          esUrlTemplate: '`<es_url>`',
+          kibanaUrlTemplate: '`<kibana_url>`',
+          configureLink: '{config.docs.beats.functionbeat}/functionbeat-configuration.html',
+        },
+      }),
+    }
+  }
+});
+
+export const createFunctionbeatCloudInstructions = () => ({
+  CONFIG: {
+    OSX_LINUX: {
+      title: i18n.translate('kbn.common.tutorials.functionbeatCloudInstructions.config.osxTitle', {
+        defaultMessage: 'Edit the configuration',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.functionbeatCloudInstructions.config.osxTextPre', {
+        defaultMessage: 'Modify {path} to set the connection information for Elastic Cloud:',
+        values: {
+          path: '`functionbeat.yml`',
+        },
+      }),
+      commands: [
+        'cloud.id: "{config.cloud.id}"',
+        'cloud.auth: "elastic:<password>"'
+      ],
+      textPost: i18n.translate('kbn.common.tutorials.functionbeatCloudInstructions.config.osxTextPost', {
+        defaultMessage: 'Where {passwordTemplate} is the password of the `elastic` user.',
+        values: { passwordTemplate: '`<password>`' },
+      }),
+    },
+    WINDOWS: {
+      title: i18n.translate('kbn.common.tutorials.functionbeatCloudInstructions.config.windowsTitle', {
+        defaultMessage: 'Edit the configuration',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.functionbeatCloudInstructions.config.windowsTextPre', {
+        defaultMessage: 'Modify {path} to set the connection information for Elastic Cloud:',
+        values: {
+          path: '`C:\\Program Files\\Functionbeat\\functionbeat.yml`',
+        },
+      }),
+      commands: [
+        'cloud.id: "{config.cloud.id}"',
+        'cloud.auth: "elastic:<password>"'
+      ],
+      textPost: i18n.translate('kbn.common.tutorials.functionbeatCloudInstructions.config.windowsTextPost', {
+        defaultMessage: 'Where {passwordTemplate} is the password of the `elastic` user.',
+        values: { passwordTemplate: '`<password>`' },
+      }),
+    }
+  }
+});
+
+export function functionbeatEnableInstructions() {
+  const defaultTitle = i18n.translate('kbn.common.tutorials.functionbeatEnableOnPremInstructions.defaultTitle', {
+    defaultMessage: 'Configure the Cloudwatch log group',
+  });
+  const defaultCommands = [
+    'functionbeat.provider.aws.functions:',
+    '  - name: fn_cloudwatch_logs',
+    '    enabled: true',
+    '    type: cloudwatch_logs',
+    '    triggers:',
+    '      - log_group_name: <cloudwatch-log-group>',
+    'functionbeat.provider.aws.deploy_bucket: <unique-bucket-name>',
+  ];
+  const defaultTextPost = i18n.translate('kbn.common.tutorials.functionbeatEnableOnPremInstructions.defaultTextPost', {
+    defaultMessage: 'Where `<cloudwatch-log-group>` is the name of the log group you want to ingest, \
+and `<unique-bucket-name>` is a valid S3 bucket name which will be used temporarily for \
```

Comment:
is the s3 bucket cleaned up after each deploy? perhaps we can say "will be used for staging the Functionbeat deployment."

Reply:
@ph ^

Reply:
I went with your wording, I think that sounds more professional than the original version.